### PR TITLE
[1.0.1] Patch to one issue, and change of styles

### PR DESCRIPTION
**Pull request by AdrianHarenczyk**

### CODE REVIEW

File: client/pom.xml [hunk 1]
```diff
@@ -3,20 +3,20 @@
 
     <modelVersion>4.0.0</modelVersion>
     <artifactId>client</artifactId>
-    <version>1.0.0</version>
+    <version>1.0.1</version>
```
Comment (gacikjakub) [inline client/pom.xml:6]: If current app version is always the same like the parent, I prefer write down <version> in properties

File: client/src/main/java/gui/controllers/PlayerSceneController.java [hunk 2]
```diff
@@ -119,13 +119,20 @@ private String getPlayerName() {
...
-        Random random = new Random();
-        int randomNumber = random.nextInt(MAX_PLAYER_SUFFIX) + 1;
         StringBuilder stringBuilder = new StringBuilder("Player");
-        stringBuilder.append(randomNumber);
+        stringBuilder.append(getRandomNumber());
         return stringBuilder.toString();
     }
 
+    private int getRandomNumber() {
+        Random random = new Random();
```
Comment (gacikjakub) [inline client/src/main/java/gui/controllers/PlayerSceneController.java:128]: I'm not sure if the object random will be better as a class field.

File: client/src/main/java/gui/controllers/PlayerSceneController.java [hunk 3]
```diff
@@ -119,13 +119,20 @@ private String getPlayerName() {
...
     private String createRandomName() {
-        Random random = new Random();
-        int randomNumber = random.nextInt(MAX_PLAYER_SUFFIX) + 1;
         StringBuilder stringBuilder = new StringBuilder("Player");
-        stringBuilder.append(randomNumber);
+        stringBuilder.append(getRandomNumber());
         return stringBuilder.toString();
     }
 
+    private int getRandomNumber() {
```
Comment (agaped) [inline client/src/main/java/gui/controllers/PlayerSceneController.java:127]: :+1:  for method extraction

**Review by agaped — APPROVED** [review]
Code looks ok

File: client/pom.xml (hunk 1 above)
Comment (AdrianHarenczyk) [reply client/pom.xml:6]: Ok, i will consider that later on. Thanks :)

File: client/src/main/java/gui/controllers/PlayerSceneController.java [hunk 4]
```diff
@@ -119,13 +119,20 @@ private String getPlayerName() {
...
+        Random random = new Random();
+        return random.nextInt(MAX_PLAYER_SUFFIX) + 1;
+    }
+
+    private int createIDForPlayer() {
+        return getRandomNumber();
```
Comment (gacikjakub) [inline client/src/main/java/gui/controllers/PlayerSceneController.java:133]: Is this adapter necessary ?

File: client/src/main/java/gui/controllers/PlayerSceneController.java (hunk 2 above)
Comment (AdrianHarenczyk) [reply client/src/main/java/gui/controllers/PlayerSceneController.java:128]: I prefer it that way. :P

File: server/pom.xml [hunk 5]
```diff
@@ -3,19 +3,19 @@
 
     <modelVersion>4.0.0</modelVersion>
     <artifactId>server</artifactId>
-    <version>1.0.0</version>
+    <version>1.0.1</version>
```
Comment (gacikjakub) [inline server/pom.xml:6]: as above

**Review by gacikjakub — APPROVED** [review]
I recommend considering my comments, but generally is OK :+1:

File: client/src/main/java/gui/controllers/PlayerSceneController.java (hunk 4 above)
Comment (AdrianHarenczyk) [reply client/src/main/java/gui/controllers/PlayerSceneController.java:133]: It's not exactly needed, but it serves as communicating intent to the developer.

**Pull request merged**